tool/net/help.txt

```diff
@@ -534,6 +534,12 @@ HOOKS
           each time redbean accepts a new client connection. If it returns
           `true`, redbean will close the connection without calling fork.
 
+  OnLogLatency(reqtimems:int,contimems:int)
```

Comment:
Milliseconds?! Redbean is at least an order of magnitude faster than that. You need at least microsecond precision because static asset serving goes 10x faster for me.

```
(stat) "/tool/net/demo/index.html" latency r: 108µs c: 242µs
```

One thing you could do is hand over these numbers as floating point in seconds. Since it's a time interval (rather than an absolute time) there will be plenty of precision in a double.

Reply:
Right, thank you for catching this typo; it was supposed to be us, so I updated the documentation.